### DISCUSSION TITLE
[백준 11399] ATM

### DISCUSSION THREAD

**Discussion by BanApp** [ghdcksgml1/Algorithm_Study#137] [1일 1알고리즘]
<img width="1166" alt="스크린샷 2022-03-11 오후 12 50 03" src="https://user-images.githubusercontent.com/93313445/157798607-4e89e6cd-7736-4783-a295-643fe16f49ac.png">

```
import sys

n = int(sys.stdin.readline())
arr = list(map(int,sys.stdin.readline().split()))

ans = [0]
arr.sort()
for i in range(n):
    ans.append(ans[i] + arr[i])

print(sum(ans))
```
최대한 빠른시간안에 해결하려면 당연히 시간이 적게 걸리는 사람부터 먼저 처리해야한다.
은행이나 AS센터 등도 가면 실제로 경험해보고 와닿는 일이다.
내가 처리 받고자 하는 일은 5분짜리인데 앞에 1시간이 걸리는 일처리를 하고 있는 사람이 있으면 화가난다.
따라서 오름차순 정렬로 정렬해주고 누적해서 더해주면 정답이다.

**Comment by ghdcksgml1** [comment]
ㅋㅋㅋㅋㅋㅋㅋㅋ
